src/main/resources/fxml/main_window.fxml

```diff
@@ -9,11 +9,9 @@
 		   fx:controller="org.cryptomator.ui.mainwindow.MainWindowController"
 		   styleClass="main-window">
 	<VBox minWidth="650">
-		<fx:include source="main_window_title.fxml" VBox.vgrow="NEVER"/>
 		<SplitPane dividerPositions="0.33" orientation="HORIZONTAL" VBox.vgrow="ALWAYS">
 			<fx:include source="vault_list.fxml" SplitPane.resizableWithParent="false"/>
 			<fx:include source="vault_detail.fxml" SplitPane.resizableWithParent="true"/>
 		</SplitPane>
 	</VBox>
-	<fx:include source="main_window_resize.fxml"/>
```

Comment:
Is `ResizeControlle` controller and the corresponding fxml, module, etc files still used somewhere else?

Same question for the window title.

Reply:
I believe there is a lot of this "window management" code that should become obsolete

Reply:
@infeo @overheadhunter

By utilizing the native title bar specific to the operating system, many of the custom window management functions are no longer necessary. Consequently, the `ResizeController` and `MainWindowTitleController`, along with their corresponding FXML files, can be removed.

The only remaining functionality related to window management is the storage of the window position and size, which has been transferred to the `MainWindowController`. If desired, this feature can also be removed.

What are your thoughts on this? Should we keep the window position and size storage feature?

Reply:
> Should we keep the window position and size storage feature?

@mindmonk Yes. Tests showed, that also with native Window bars, the window position is not saved without those settings. Hence, we need to keep those settings.